src/style.css

```diff
@@ -51,7 +51,7 @@ body {
   margin: 0;
 }
 
-.search-box--small input[type="search"] {
+.search-box--small .search-box-field {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element